apps/docs/content/components/Pillar/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Pillar
+description: Use pillars to group related content together.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image showing the Pillar anatomy with an icon, a heading and a description in a vertical order.](https://github.com/primer/brand/assets/912236/c52debd2-4277-42c9-9a73-bd6bd087dd1e)
+
+- Icon: It provides a visual cue to the user about the content of the piller. Icon is optional.
+- Heading: The main title of the pillar. It should be short and concise.
```

Comment:
Should pillar be capitalized as it's the component name? We do this with all other components, like Grid below so I think we should do that here too.

```suggestion
- Heading: The main title of the Pillar. It should be short and concise.
```